cdci_data_analysis/configurer.py

```diff
@@ -250,7 +250,7 @@ def __init__(self, cfg_dict, origin=None):
                                      disp_dict.get('product_gallery_options', {}).get('converttime_revnum_service_url', 'https://www.astro.unige.ch/mmoda/dispatch-data/gw/timesystem/api/v1.0/converttime/UTC/{}/REVNUM'),
                                      disp_dict.get('renku_options', {}).get('renku_gitlab_repository_url', None),
                                      disp_dict.get('renku_options', {}).get('renku_base_project_url', None),
-                                     disp_dict.get('renku_options', {}).get('ssh_key_path', None)
+                                     disp_dict.get('renku_options', {}).get('ssh_key_path', None),
```

Comment:
did you try to add option for ontology? we should probably.

Reply:
I tried, then reverted.  
This functionality is used only in nb2workflow plugin, so I added the option there. This requires less changes than passing this option all the way from flask app up to Queries (which are anyway initialized in the plugin code).